app/views/locations/_form.html.erb

```diff
@@ -176,22 +176,24 @@
     });
 
   })();
-  
   function saveLocation() {
     var tags = 'lat:' + blurredLocation.getLat() + ',lon:' + blurredLocation.getLon() + ',zoom:' + blurredLocation.getZoom();
     if (blurredLocation.isBlurred()) tags = tags + ',' + 'location:blurred'
     if ($('#placenameDisplay').val() !== ('Location unavailable' || '')) {
       tags = tags + ',' + 'place:' + parameterize($('#placenameDisplay').val());
     }
     <% if params[:url] %>
-      addTag(tags, '<%= params[:url] %>');
+      addTag(tags, '<%= params[:url] %>', "", locationCallback);
     <% else %>
-      addTag(tags);
+      addTag(tags, null, "", locationCallback);
     <% end %>
-    
     <% if params[:reload] %>setTimeout(function() { location.reload(); }, 3000);<% end %>
   }
 
+  function locationCallback(event){
+    notyNotification('relax', 3000, 'success', 'topCenter', 'Location Saved');
```

Comment:
Hey from the screenshot, `topCenter ` looked different. @jywarren can you verify if should be `topRight` maybe?